edx_solutions_api_integration/imports/views.py

```diff
@@ -224,7 +224,10 @@ def _enroll_user(self, data, errors, response={}):
                 # Add permission for role.
                 permission = permissions[role]
                 permission_groups = Group.objects.get(groupprofile__name=permission)
-                permission_groups.user_set.add(user.id)
+                permission_groups.user_set.get_or_create(id=user.id)
+        except IntegrityError:
+            # This situation can occur if one user try to enroll in multiple courses
```

Comment:
@ahmed-zubair12 Instead of comment and `pass`, can we log relevant info here so in case an error occurs it is traceable through splunk logs?